### ISSUE TITLE
Python.EXE will not open for simulator from installer

### ISSUE THREAD

**Issue by dalevens** [dalevens/OpenTitration#16]
Simulator cannot find the python.exe file. Switching the file does not resolve the issue.

This can be for two reasons:

1. the file structure does not work properly
2. Administrator privilege's are required for it to run properly

<img width="1310" alt="python path not found error" src="https://user-images.githubusercontent.com/64719450/113456764-6231b100-93cb-11eb-8f5b-3a890f0f6560.png">

**Comment by jkelowitt** [comment]
![image](https://user-images.githubusercontent.com/58435724/113459866-12a3b300-93d4-11eb-853f-5c753ad00721.png)

It looks like the program is looking for that directory no matter what the custom set directory is in settings. 

I feel like there is a hard coded python directory somewhere messing this up.

**Comment by dalevens** [comment]
There were some errors hard coded with the directory picker. However and unfortunately that does not appear to be the problem. I placed a message flag that would tell me exactly what directory it goes too. I will investigate further whether it is indeed an issue on my end. Updates to follow.

<img width="704" alt="pythondirectoryconfirmation" src="https://user-images.githubusercontent.com/64719450/113465410-efd3c780-93f0-11eb-8817-227d0eaa00c3.png">

**Comment by dalevens** [comment]
Further efforts into looking into the problem have yielded new and interesting results. Firstly I doubled check to make sure that C# was sending the correct directory to python for the 'script.py'. I get this

![image](https://user-images.githubusercontent.com/64719450/114237528-6ec08700-9940-11eb-9fd8-17dd8fdd5d0c.png)

This appears to be correct, so I continued to do a little digging.


It appears that this may be an issue with the config file. Known as pyvenv.cfg the default setting is to look into the python.exe file at C:\Program Files (x86)\Microsoft Visual Studio\Shared\Python37_64.

![image](https://user-images.githubusercontent.com/64719450/114236548-fefdcc80-993e-11eb-9396-8163962acb00.png)

Changing this to the OpenTitration directory of C:\Program Files\OpenTitrationFoundation\OpenTitration\python_env fixes this problem. However, python is unable to find script.py and titration_simulator. Instead it returns a blank error message, indicating that python simply ran until it timed out, returning no error. For whatever reason it is unable to find the scripts under these conditions. 

![image](https://user-images.githubusercontent.com/64719450/114237056-c4e0fa80-993f-11eb-87ef-23cc471dae62.png)

This error message is supposed to open up and tell you what error python has returned if no values have been returned by the python script. Because the python script never runs it times out and closes without returning any numbers. 

**Possible Solution**

The directory will be static at C:\Program Files\OpenTitrationFoundation\OpenTitration\ when installed by the installation package. Would it be possible to simply have everything open up from there?
